src/recipe.cpp

```diff
@@ -1216,14 +1216,15 @@ bool recipe::hot_result() const
 
 int recipe::makes_amount() const
 {
-    int makes = 0;
+    int makes_charges;
```

Comment:
You should just initialize this to 1 and remove the else case.

Reply:
I thought this is more clear

Reply:
Ok, did it your way :)